integrations/azure_ai_search/.gitignore

```diff
@@ -0,0 +1,163 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
I would say that this file is not needed: we already have [this](https://github.com/deepset-ai/haystack-core-integrations/blob/e97cfee4c2b0868d96e4a4a335ec5cd7ee0f6cf1/.gitignore) at the root level